lib/matplotlib/bezier.py

```diff
@@ -219,6 +220,114 @@ def point_at_t(self, t):
         """Evaluate curve at a single point *t*. Returns a Tuple[float*d]."""
         return tuple(self(t))
 
+    @property
+    def arc_area(self):
+        r"""
+        Signed area swept out by ray from origin to curve.
+
+        Counterclockwise area is counted as positive, and clockwise area as
+        negative.
+
+        The sum of this function for each Bezier curve in a Path will give the
+        signed area enclosed by the Path.
+
+        Returns
+        -------
+        float
+            The signed area of the arc swept out by the curve.
+
+        Notes
+        -----
+        A simple, analytical formula is possible for arbitrary bezier curves.
+
+        Given a bezier curve B(t), in order to calculate the area of the arc
+        swept out by the ray from the origin to the curve, we simply need to
```

Comment:
That's a matter of perspective...
```suggestion
        swept out by the ray from the origin to the curve, we need to
```